### DISCUSSION TITLE
Welcome to ActivityPods discussions!

### DISCUSSION THREAD

**Discussion by srosset81** [assemblee-virtuelle/activitypods#131] [Announcements]
## 👋 Welcome!
  We’re using GitHub Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CartyBoston** [comment]
Sebastien and team Hi,

My name is Carty Castaldi, I have been lurking here for a while, have learned a ton, and am hoping I can begin to contribute.

My journey started with Mastodon and Activitypub, I brought up a Mastodon instance ~a year ago then dived into ActivityPub, instrumenting my Mastodon instance to trace messages.

I have been in the [industry for a long time](https://www.linkedin.com/in/carty-castaldi-08a669/), most recently assembling and leading sw product development teams in greater Boston. I hadn't been coding much commercially but personally have built some web apps with Python/Flask. I am reasonably familiar with linux.

Much of your environment is new to me, I had not done any coding with JavaScript outside a browser. Node, React, and even Docker are new to me, as I said I have learned a ton, deliberately taking the time to dive into each tool.

I work on a Debian instance at Google Cloud Platform, I have the backend and frontend running locally. I am having some issues with deploying to the outside world, I'll cover that in a separate note.

Team, I was around in the days when TCP/IP was being standardized, the notion that machines could talk to each other - could interoperate - was revolutionary and very exciting. I can see that ActivityPub is the same revolutionary opportunity at the application layer. But ActivityPub is messaging, how/where do we store the (decentralized) data? I found MIT Solid, and here I am.

I am looking forward to working with you all, I can see you are doing important work.

-Carty
carty@roundpond.net
@CartyBoston@mastodon.roundpond.net
